### PR TITLE
5.6 connect available toppings to the menu items in backend

### PR DESCRIPTION
**Pull request by steget**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ app.use(bodyParser.json());
 
 // hashmap with orders
 const orderMap = new Map();
+const menuItemMap = new Map();
```
Comment (amandadehlen) [inline server.js:24]: When you use this but do not change the whole backend representation of menu item to use this, it do not add value /or problematic.

Comment (steget) [reply server.js:24]: #80

File: server.js [hunk 2]
```diff
@@ -203,6 +213,39 @@ function getNewItemId() {
     return latestItemId;
 }
 
+function initMenuItems() {
+    //query db and replace with this instead
+    //db.getMenuItems(insertMenuItems);
+    insertMenuItems(menuItemArray)
```
Comment (Pear99) [inline server.js:219]: This function does not gather any data from the database even though I have it set up.

File: server.js [hunk 3]
```diff
@@ -203,6 +213,39 @@ function getNewItemId() {
     return latestItemId;
 }
 
+function initMenuItems() {
+    //query db and replace with this instead
+    //db.getMenuItems(insertMenuItems);
```
Comment (Pear99) [inline server.js:218]: When I uncommented and ran this line it said there is no function from db called getMenuItems

**Comment by steget** [comment]
@Pear99 The rows you highlighted are supposed to be commented. The implementation of getting menuItems from db is done in task 5.5.6. Look #77

**Pull request merged**